docs/operator-manual/applicationset.yaml

```diff
@@ -3,36 +3,312 @@ kind: ApplicationSet
 metadata:
   name: test-hello-world-appset
   namespace: argocd
+  # To preserve this annotation and label we can use the preservedFields property 
+  preservedFields:
+    # This annotation and label exists only on this Application, and not in 
+    # the parent ApplicationSet template:
+    # ignoreApplicationDifferences is the preferred way to accomplish this now.
+    annotations: 
+      my-custom-annotation: some-value
+    labels:
+      my-custom-label: some-value
+
```

Comment:
idk how this has changed, but preservedFields is currently under spec, not metadata, and it takes a list of strings.

Reply:
Good catch. Can you open a PR to fix this?

Reply:
https://github.com/argoproj/argo-cd/pull/20206